util/confutil/config.go

```diff
@@ -151,7 +152,11 @@ func LoadConfigTree(fp string) (*toml.Tree, error) {
 	defer f.Close()
 	t, err := toml.LoadReader(f)
 	if err != nil {
-		return t, errors.Wrap(err, "failed to parse config")
+		return t, errors.Wrap(err, "failed to parse buildkit config")
+	}
```

Comment:
Wondering; will this be problematic when crating a builder using an older version of buildkit? (i.e. would validation only work correctly if the version matches the one that's vendored?)

Reply:
I don't think this will be an issue as we are backward compat. Also we are already using this logic to get the registry config in https://github.com/docker/buildx/blob/567361d4948b06c559b41797c58e683f9a587583/store/storeutil/storeutil.go#L125-L128

Reply:
Ah yeah, I wasn't sure, so thought  I'd mention it. I recall that I ran into changes in the garbage-collect configuration that changed, and for which I needed to make changes in moby/moby, because it used those; https://github.com/moby/moby/pull/48634#issuecomment-2407072949